decidim-core/db/migrate/20240529161054_add_link_to_attachments.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddLinkToAttachments < ActiveRecord::Migration[7.0]
+  def change
+    add_column :decidim_attachments, :link, :string
+  end
```

Comment:
@andreslucena I opened this draft PR to show the path I'm following because at the end I saw that reusing the existing `Attachment` model for links was simpler than I thought. 

The model already supported having no `file` and the changes I had to do to the Create/Update Attachment Commands and the Attachment Form and Modelare are minimal. 

There are few thing left but I believe that the big changes are already in this draft.

cc @entantoencuanto